src/query/procedure/mg_procedure_impl.cpp

```diff
@@ -1497,9 +1542,15 @@ mgp_error mgp_properties_iterator_next(mgp_properties_iterator *it, mgp_property
           return nullptr;
         }
         memgraph::utils::OnScopeExit clean_up([it] { it->current = std::nullopt; });
-        it->current.emplace(memgraph::utils::pmr::string(it->graph->impl->PropertyToName(it->current_it->first),
-                                                         it->GetMemoryResource()),
-                            mgp_value(it->current_it->second, it->GetMemoryResource()));
+        auto propToName = std::visit(
+            memgraph::utils::Overloaded{
```

Comment:
Remove `memgraph::utils::Overloaded`. It's not needed.

Reply:
Done